### ISSUE TITLE
Mod doesn't support note_block variations

### ISSUE THREAD

**Issue by Andre601** [Sollace/Presence-Footsteps#108]
#### Description
I've made a resource pack containing a `blockmap.json` in order to change the sound used for specific blocks, in my case note blocks with specific instrument and note values, but the mod doesn't seem to recognize that the resource pack even exists as it still uses the default sounds for note blocks (woodutility).

The resource pack itself is considered a valid one by the MC client (shows in the resource pack window) and can be applied.

I've tried changing the order of the resource pack to have it at the top of the list and at the very bottom (below the Vanilla internal textures) but no changes happened here.

#### Reproduce
1. Create a resource pack only containing a `pack.mcmeta` and `assets/presencefootsteps/config/blockmap.json`
2. Add the following values to the `blockmap.json`: https://paste.helpch.at/zisibitogi.json
3. Make a zip out of the files and apply it as a resource pack to the client.
4. Test it out on various note blocks (i.e. in Minecraft's Debug Worldtype)

#### Expected behaviour
Presence Footsteps should take the values from the resource pack and apply it to the specific blocks.

#### Actual behaviour
The mod seems to ignore or intentionally not load the `blockmap.json` file.

#### Additional Info
- Minecraft Client: 1.17.1
- Fabric Loader: 0.11.7-1.17.1
- Loaded Mods: 3d Skin Layers, FirstpersonModel, Illuminations, Indium, Inventory HUD +, Iris, LambDynamicLights, Mod Menu, NotEnoughAnimations, NTCM, Origins Enhanced, Presence Footsteps, Shulker Box Tooltip, Sodium, WaveyCapes

**Comment by Andre601** [comment]
Made some other testing including printing the blockstate of the block the player/entity is on and it gave the following output for note blocks:
```
Block{minecraft:note_block}[instrument=harp,note=1,powered=false]
```

I thought adding the missing `powered=false` state to the entries would solve this, but it doesn't unfortunately.
I also tried other tricks such as reordering the list and removing the basic `"minecraft:note_block": "woodutility"` entry from the JSON, but this didn't help either.

So in conclusion can I now 100% eliminate the possibility that this is an error on my end with the blockmap.json and is instead 100% a bug on the mod's end which should be solved.

I know you probably have other things that need your attention, but I would appreciate to at least get an acknowledgement of sorts that you saw and read those messages, because at this point do I feel like as if you give a shit about the issues here...

**Comment by Sollace** [comment]
Duplicate of #107

**Comment by Andre601** [comment]
Considering my original issue is closed and that I mentioned I open this one for better tracking (Since closed issues aren't really good for tracking active bugs imo) would I not quite count this as a duplicate, but doesn't really matter.

Glad a fix was found.

**Comment by Andre601** [comment]
Quickly tested out 54719f0 by compiling it, but no fix yet. The resource pack and direct modifications to the mod itself are still completely ignored here and the note blocks still emit the `woodutility` sound as if not changes were made at all.

**Comment by Andre601** [comment]
I noticed a small oddity that I want to mention.

If you're walking right at the edge of a noteblock and another block (in my case moss) will the mod play the `woodutility` sound as if the noteblock was the default (Wasn't covered by the RP itself, which in my case, it is.)

Not sure why this happens and it's a rather rare and minor issue. Other than that does everything else work smoothly. Currently use a own build version from the 1.17 branch and I hope there will be a release soon.

I close this issue now since the main points of it have been fixed. Thanks for the work.
